### PR TITLE
feat: Add disk input motherboard network and dprocess interface

### PR DESCRIPTION
**Pull request by wyu71**
Add disk input motherboard network and dprocess interface

Log: disk input motherboard network and dprocess interface

### CODE REVIEW

File: dtklsdevice/include/ddiskdevice.h [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+class DDiskDevicePrivate;
+class DDiskDevice : public QObject
+{
+public:
+    explicit DDiskDevice(QObject *parent = nullptr);
+    ~DDiskDevice();
+
+    int count();
+    QString vendor(const int &index);
```
[the remark this answers is not in the document]
Comment (wyu71) [reply dtklsdevice/include/ddiskdevice.h]: 已安建议修改

**Comment by toberyan** [comment]
+1

**Comment by Groveer** [comment]
+1

**Review by uniontechzhangsong — COMMENTED** [review]
+1

**Review by toberyan — APPROVED** [review]

**Pull request merged**